OpenXmlPowerTools.Tests/OpenXmlPowerTools.Tests.csproj

```diff
@@ -1,28 +1,24 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net452;net46</TargetFrameworks>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
     <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
+    <TargetFrameworks>net461;netcoreapp2.0</TargetFrameworks>
```

Comment:
Why did you change the target frameworks? Let's keep net452 in the list as well - this allows testing the .net 4.5 build